internal/pkg/log/log.go

```diff
@@ -67,4 +68,5 @@ type contextLogger interface {
 type withAttributes interface {
 	With(attrs ...attribute.KeyValue) Logger
 	WithComponent(component string) Logger
+	WithDuration(v time.Duration) Logger
```

Comment:
`logger.WithDuration().Info()`, I don't have a better idea, because we cannot add more arguments to `Infof`, and we need to cover both: `Info/Infof`.

Reply:
Example: 
![image](https://github.com/keboola/keboola-as-code/assets/19371734/c371b65c-bcf7-424d-aef5-afd61ee98eac)

Reply:
2. In the old code above the is no `clock` instance, and for duration measurement it is OK.

Reply:
`ctxattr` are greate for common attributes, for span + log, but here it is an overkill, isn't it?

```
logger.WithDuration(time.Since(startTime)).Infof(ctx, `loaded Storage API index with "%d" components`, len(indexWithComponents.Components))
```

vs

```
logger.Infof(
    ctxattr.ContextWith(ctx, attribute.String("duration", time.Since(startTime).String()),
    `loaded Storage API index with "%d" components`,
    len(indexWithComponents.Components),
)
```

Reply:
Standup discussion notes:
```
1. logger.WithDuration(time.Since(startTime))
2. logger.With(attribute.String("duration", time....)).Infof(template, arg1, arg2)
3. logger.Info(fmt.Sprintf(template, arg1, arg2), ... attribute.String("duration", time....))
```

slog?